### PR TITLE
PyTorch experiment tracking blog post

### PR DESCRIPTION
**Pull request by fa9r**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Briefly describe the changes you have introduced.

### CODE REVIEW

File: _posts/2022-10-27-pytorch-experiment-tracking.md [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+        train=False,
+        download=True,
+        transform=ToTensor(),
+    )
+
+    # Create data loaders.
+    train_dataloader = DataLoader(training_data, batch_size=batch_size)
+    test_dataloader = DataLoader(test_data, batch_size=batch_size)
+
+    return train_dataloader, test_dataloader
```
Comment (htahir1) [inline _posts/2022-10-27-pytorch-experiment-tracking.md:284]: Generally speaking, is returning dataloaders (which are lazy objects) the best way to showcase? Im still unsure about this part in ZenML

**Review by htahir1 — COMMENTED** [review]
Great work! Left a few comments!

File: _posts/2022-10-27-pytorch-experiment-tracking.md [hunk 2]
```diff
@@ -0,0 +1,657 @@
...
+Since we will be using W&B in our pipeline, make sure to create an account at the official [site](https://wandb.ai/home). It's free to get started. Next, create a project and get the entity, project name and the API key.
+
+Now with those details, let's put them in our code and start running them, shall we?
+
+Of course not.
+
+Sharing access credentials in your codes or files is a quick way to set your butt on fire.
+I wouldn't recommend it.
+
+![butt-on-fire](../assets/posts/pytorch_wandb/soccer-butt.gif)
```
[the remark this answers is not in the document]
Comment (dnth) [reply _posts/2022-10-27-pytorch-experiment-tracking.md:447]: I'll change this one if it's too offensive :P

File: _posts/2022-10-27-pytorch-experiment-tracking.md (hunk 1 above)
Comment (htahir1) [reply _posts/2022-10-27-pytorch-experiment-tracking.md:284]: Ok then lets go for it!

**Review by htahir1 — APPROVED** [review]
LGTM except we can now delete `assets/posts/pytorch_wandb/soccer-butt.gif` :-)

**Pull request merged**